utils/rpms/daos.spec

```diff
@@ -219,8 +219,10 @@ Requires: lbzip2
 Requires: attr
 %if (0%{?suse_version} >= 1315)
 Requires: lua-lmod
+Requires: libcapstone-devel
```

Comment:
It's a requirement for running the test - which builds from src so I think it's a Requires for this end.  It's also a BuildRequires as well but that should already be in place.

Reply:
That's kind of weird though.  We are probably the only people that will run that test.  Seems like this should go into post provisioning script or some such.

Reply:
> That's kind of weird though. We are probably the only people that will run that test. Seems like this should go into post provisioning script or some such.

I am trying to eliminate the [external dependency knowledge](https://github.com/daos-stack/daos/blob/master/ci/functional/required_packages.sh) so I would rather we do not add to it.

Any packages that are needed to run tests should be specified as dependencies of the test RPM that they are packaged in.  People trying to run the tests RPMs should not have to go hunting through our CI setup scripts to find the things that the tests in the RPMs need.

Ultimately, because we have decided to make building DAOS a test of DAOS, that means we have transferred our DAOS build dependencies into test dependencies.